src/redux/actions.js

```diff
@@ -0,0 +1,9 @@
+import * as types from './actionTypes';
+import { isDisplayable } from '../pressButtonLogic';
+
+export function getType(value) {
+    if (isDisplayable(value))
+        return { type: types.DISPLAY_NUMBER, value: value };
```

Comment:
You don't need an action for this. The display should update itself when the state changes accordingly.